pyvista/utilities/helpers.py

```diff
@@ -1458,3 +1459,56 @@ def cubemap(path='', prefix='', ext='.jpg'):
         texture.SetInputDataObject(i, flip.GetOutput())
 
     return texture
+
+
+def set_default_active_vectors(mesh: 'pyvista.DataSet') -> None:
```

Comment:
Since we have `import pyvista` at the top this can be the actual type, right?
```suggestion
def set_default_active_vectors(mesh: pyvista.DataSet) -> None:
```

Reply:
This is what I did first and it leads to a circular import in a way I don't understand. This might only be true for earlier python versions, e.g. 3.7.  I could try using `TYPE_CHECKING` and `from pyvista import DataSet` if this preferred.

Reply:
Hmm, I guess it's that there's still a circular import, but the partially initialised module doesn't cause a problem because we aren't accessing any of the undefined names here. And using `pyvista.DataSet` eagerly violates that.

There's nothing wrong with the string annotation, I just thought we don't need it stringified. Let's leave it like this then, and maybe remove all the string annotations in the distant future when lazy evaluation is the default for all supported Python versions.